### DISCUSSION TITLE
Arbitrary values for background-position and background-size - is it possible?

### DISCUSSION THREAD

**Discussion by fabriciojs** [tailwindlabs/tailwindcss#4626] [Help]
I could not manage to use custom values for bg size/position with the `bg-[]` syntax, and didn't find an alternative (only writing the variations at tailwind.config of course, which is not ideal/scalable).

Is it possible at all at the moment? Or will it only work for bg colors?

**Comment by MuttakinHasib** [comment]
hey @adamwathan  please help

**Reply by adamwathan** [reply]
Won’t work for those utilities yet but planning to make it possible in the next couple of months. In the mean time I would just add values to your config or use inline styles.
